packages/cardpay-reward-programs/cardpay_reward_programs/main.py

```diff
@@ -13,34 +14,19 @@
 load_dotenv()
 
 cached_client = S3Client(
-    local_cache_dir="mycache",
+    local_cache_dir=".cache",
     boto3_session=Session(
-        aws_access_key_id=os.getenv("AWS_ACCESS_KEY_ID"),
-        aws_secret_access_key=os.getenv("AWS_SECRET_ACCESS_KEY"),
     ),
 )
 cached_client.set_as_default_client()
 
 
-def select_rule(name, core_parameters, user_defined_parameters):
-    rule_constructor = globals()[name]
-    instance = rule_constructor(core_parameters, user_defined_parameters)
-    return instance
-
-
-def to_rule(parameters):
-    core_parameters, user_defined_parameters = get_parameters(parameters)
-    name = to_camel_case(core_parameters["docker_image"])
```

Comment:
We can't use the docker image here as 

1. It'll have a different format (start acct_id.useast1.ecr/etc) rather than just "MinSpend"
2. The code here shouldn't know what docker image it's in, it's just run.

Reply:
ok Agree. The `docker_image` is not the correct key to dictate the `rule_name`. 

Note: I excluded the `rule_name` from the `parameters.json`. Shall it be re-inserted into `run`. I don't have strong opinion about this.